src/components/Button.tsx

```diff
@@ -204,4 +211,11 @@ const iconStyles: Record<ButtonSize, IconProps["xss"]> = {
 };
 
 export type ButtonSize = "sm" | "md" | "lg";
-export type ButtonVariant = "primary" | "secondary" | "tertiary" | "tertiaryDanger" | "danger" | "text";
+export type ButtonVariant =
+  | "primary"
+  | "secondary"
+  | "tertiary"
+  | "tertiaryDanger"
+  | "danger"
+  | "text"
+  | "secondaryText";
```

Comment:
**thought/nit:** Maybe `textSecondary`? Just to be inline with how `tertiaryDanger` is the "danger" version of our "tertiary" button. It appears that this one is the "secondary" version of our "text" button vs a "text" version of our "secondary" button.

Reply:
I'm realizing that this whole "it's a x of y vs y of x" thought doesn't really make sense. Good how it is!